CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## Unreleased
 
+## 0.17.0 - 2023-03028
+
+#### Breaking Changes
+
+- `wayland-rs` dependencies are updated to 0.30 and all APIs have changed significantly as a result
+
+#### Additions
+
+- `xkbcommon` is now optional and gated by the `xkbcommon` feature
```

Comment:
I recall 0.16 having libxkbcommon be dlopened. The `xkbcommon` crate does not do so it's a required dependency if the feature is enabled